turtlebot_concert/secret_file_users

```diff
@@ -0,0 +1,3 @@
+jchoclin@creativa77.com 312b8782b4196158fb07f184f11b63b1
```

Comment:
It would be nice to also add Jihoon's and Huey's e-mail address, and make the password something obvious and known, so that they can run the demo and try with different users.